midAssessment/factorial.rb

```diff
@@ -0,0 +1,17 @@
+# Write a program to calculate the factorial of a given number.
+print "Enter a number: "
```

Comment:
```suggestion
print 'Enter a number: '
```

Use single quotes. Double are preferrable in interpolation only.

Reply:
Correct at all places.

Reply:
Corrected at all places